docs/ROADMAP.md

```diff
@@ -66,6 +66,7 @@ To discuss the roadmap and the related initiatives, use the
 * More DCL features
   * `Map<K, V>` properties
   * Container defaults
+* Reference Version Catalogs for dependency declarations in DCL
```

Comment:
We should update the roadmap holistically/separate from these changes.  We already know that we're dropping container defaults from EAP3 @eskatos

Reply:
OK, I'll remove this from this PR.